src/components/ha-stt-picker.ts

```diff
@@ -41,13 +42,32 @@ export class HaSTTPicker extends LitElement {
     if (!this._engines) {
       return nothing;
     }
-    const value =
-      this.value ??
-      (this.required
-        ? this._engines.find(
-            (engine) => engine.supported_languages?.length !== 0
-          )
-        : NONE);
+
+    let value = this.value;
+    if (!value && this.required) {
+      for (const entity of Object.values(this.hass.entities)) {
```

Comment:
Do we really have to go through all entities? Can't this be added to the `stt/engine/list` command?

Reply:
I didn't want to expand that feature, especially as core wants to remove the default methods (they are not being used otherwise)